l1-contracts/src/core/libraries/rollup/EpochProofLib.sol

```diff
@@ -99,25 +100,20 @@ library EpochProofLib {
   ) internal view returns (bytes32[] memory) {
     RollupStore storage rollupStore = STFLib.getStorage();
 
+    require(_args.endArchive != bytes32(0), Errors.Rollup__EndArchiveIsZero());
+
     // TODO(#7373): Public inputs are not fully verified
 
     {
       // We do it this way to provide better error messages than passing along the storage values
       {
         bytes32 expectedPreviousArchive = rollupStore.archives[_start - 1];
+        require(expectedPreviousArchive != bytes32(0), Errors.Rollup__PreviousArchiveIsZero());
```

Comment:
Question: what should this be for 1st Epoch? `GENESIS_ARCHIVE_ROOT`?

Reply:
Yep. It should be the genesis for the first one.